ament_uncrustify/ament_uncrustify/main.py

```diff
@@ -72,6 +77,20 @@ def main(argv=sys.argv[1:]):
         print("Could not config file '%s'" % args.config_file, file=sys.stderr)
         return 1
 
+    if args.linelength is not None:
+        # check if different from config file
+        config = ConfigParser()
+        with open(args.config_file, 'r') as h:
+            config_str = h.read()
+        config.read_string('[DEFAULT]\n' + config_str)
+        code_width = config['DEFAULT']['code_width']
+        code_width = int(re.split('[ \t#]', code_width, maxsplit=1)[0])
+        if args.linelength != code_width:
+            # generate temporary config file with custom line length
+            temp_config = tempfile.NamedTemporaryFile('w')
+            temp_config.write(config_str + '\ncode_width=%d' % args.linelength)
```

Comment:
This appends the config parameter, but does not remove the original if it's already set. Will that work correctly in that case?